src/modules/Servicehosting/Service.php

```diff
@@ -123,13 +123,16 @@ public function action_activate(\Model_ClientOrder $order)
 
         $model->username = $username;
         $model->pass = $pass;
-        $this->di['db']->store($model);
 
         if (!isset($c['import']) || !$c['import']) {
             [$adapter, $account] = $this->_getAM($model);
             $adapter->createAccount($account);
         }
 
+        $model->pass = '*******';
```

Comment:
Why is this not up a few lines with the rest of the code related to the model?

Reply:
$model is also used to create the hosting account in 

[$adapter, $account] = $this->_getAM($model);
            $adapter->createAccount($account);

If we set it to ***** and then create the account all user passwords will be the same. 

Ideally we should drop the column for password and forget about it saving it...

Reply:
Dropping the column entirely is an easy enough task

Reply:
Then we should do that imo